tensorboard/plugins/pr_curve/pr_curves_plugin_test.py

```diff
@@ -57,23 +57,45 @@ def validatePrCurveEntry(
       expected_step,
       expected_precision,
       expected_recall,
+      expected_true_positives,
+      expected_false_positives,
+      expected_true_negatives,
+      expected_false_negatives,
+      expected_thresholds,
       pr_curve_entry):
     """Checks that the values stored within a tensor are correct.
 
     Args:
       expected_step: The expected step.
-      expected_precision: A list of float precision values.
-      expected_recall: A list of float recall values.
+      expected_precision: A list of float values.
+      expected_recall: A list of float values.
+      expected_true_positives: A list of int values.
+      expected_false_positives: A list of int values.
+      expected_true_negatives: A list of int values.
+      expected_false_negatives: A list of int values.
+      expected_thresholds: A list of floats ranging from 0 to 1.
       pr_curve_entry: The PR curve entry to evaluate.
     """
     self.assertEqual(expected_step, pr_curve_entry['step'])
     # We use an absolute error instead of a relative one because the expected
     # values are small. The default relative error (rtol) of 1e-7 yields many
     # undesired test failures.
+    atol = 1e-7
```

Comment:
[optional] Here's how I would do it in the global scope:

```py
assert_allclose = functools.partial(np.testing.assert_allclose, rtol=0, atol=1e-7)
```

Reply:
I really like that idea!

Reply:
Done.

Reply:
😃